### PR TITLE
NIAD-459: Create OpenApi specification for POST SCR API

### PR DESCRIPTION
**Pull request by Broom94**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/summary-care-record.yaml [hunk 1]
```diff
@@ -38,43 +38,49 @@ paths:
...
       responses:
         "200":
           description: OK successful
           content:
             application/fhir+json:
               schema:
                 type: object
+              example: "<OperationOutcome xmlns=\"http://hl7.org/fhir\">\n    <issue>\n        <severity value=\"information\"></severity>\n        <code value=\"information\"></code>\n        <diagnostics value=\"https://fhir.nhs.uk/STU3/ValueSet/Spine-ErrorOrWarningCode-1 == static display value\"></diagnostics>\n    </issue>\n</OperationOutcome>"
```
Comment (michael-kainos) [inline specification/summary-care-record.yaml:49]: Shouldn't these be in JSON?

File: specification/summary-care-record.yaml [hunk 2]
```diff
@@ -38,43 +38,49 @@ paths:
...
+        "400":
+          description: ACL consent not set
+          content:
+            application/fhir+json:
+              schema:
+                type: object
+              example: "<OperationOutcome xmlns=\"http://hl7.org/fhir\">\n    <issue>\n        <severity value=\"error\"></severity>\n        <code value=\"invalid\"></code>\n        <details>\n            <text value=\"https://fhir.nhs.uk/STU3/ValueSet/Spine-ErrorOrWarningCode-1 == static display value\"></text>\n        </details>\n    </issue>\n</OperationOutcome>"
```
Comment (michael-kainos) [inline specification/summary-care-record.yaml:56]: Here too?

**Review by michael-kainos — CHANGES_REQUESTED** [review]

File: specification/summary-care-record.yaml [hunk 3]
```diff
@@ -38,43 +38,49 @@ paths:
...
       responses:
         "200":
           description: OK successful
           content:
             application/fhir+json:
               schema:
                 type: object
+              example: "{\n    \"OperationOutcome\": {\n      \"issue\": {\n        \"severity\": {\n            \"value\": \"information\"\n        },\n        \"code\": {\n            \"value\": \"informational\"\n        },\n        \"diagnostics\": {\n            \"value\": \"Resource has been successfully updated.\"\n        }\n      }\n    }\n}"
+        "400":
```
Comment (michael-kainos) [inline specification/summary-care-record.yaml:50]: This 400 response is the same as the 412?

File: specification/summary-care-record.yaml [hunk 4]
```diff
@@ -38,43 +38,49 @@ paths:
...
         "412":
           description: ACL consent not set
           content:
             application/fhir+json:
               schema:
-                type: object                
-        "504":
-          description: Gateway Timeout
+                type: object
+              example: "{\n    \"OperationOutcome\": {\n      \"issue\": {\n        \"severity\": {\n            \"value\": \"error\"\n        },\n        \"code\": {\n            \"value\": \"invalid\"\n        },\n        \"diagnostics\": {\n            \"value\": \"ttps://fhir.nhs.uk/STU3/ValueSet/Spine-ErrorOrWarningCode-1 == static display value\"\n        }\n      }\n    }\n}"
```
Comment (michael-kainos) [inline specification/summary-care-record.yaml:63]: Use diagnostics value to provide error descriptions of what will be returned

**Review by michael-kainos — CHANGES_REQUESTED** [review]

File: specification/summary-care-record.yaml (hunk 3 above)
Comment (Broom94) [reply specification/summary-care-record.yaml:50]: yes correct, please see diagnostic value for error (link to enums)

File: specification/summary-care-record.yaml (hunk 4 above)
Comment (Broom94) [reply specification/summary-care-record.yaml:63]: it is used, can have a quick call to discuss?

**Review by michael-kainos — APPROVED** [review]

**Pull request merged**